pkg/types/validation/installconfig.go

```diff
@@ -149,6 +150,19 @@ func ValidateInstallConfig(c *types.InstallConfig) field.ErrorList {
 	return allErrs
 }
 
+func fixAMD64ArchSupport(controlPlane *types.MachinePool, compute []types.MachinePool) {
+	// "AMD64" is the name chosen by AMD for their 64-bit extension to the Intel x86 instruction set.
+	//  Before release, it was called "x86-64" or "x86_64", and some distributions still use these names
+	if controlPlane != nil && (controlPlane.Architecture == "x86-64" || controlPlane.Architecture == "x86_64") {
+		controlPlane.Architecture = types.ArchitectureAMD64
+	}
+	for i, c := range compute {
+		if c.Architecture == "x86-64" || c.Architecture == "x86_64" {
```

Comment:
I'm not sure we need this translation. In the docs (e.g. https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal/installing-bare-metal.html#installation-configuration-parameters-optional_installing-bare-metal) it appears we are pretty clear that the valid value is `amd64`, and we validate this [here](https://github.com/openshift/installer/blob/master/pkg/types/validation/machinepools.go#L74-L76).